### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by naftalimurgor**
Update the documentation as per this issue https://github.com/aromalanil/react-rhino/issues/1
Suggestions are welcome :confetti_ball: 
Thanks

### CODE REVIEW

**Comment by naftalimurgor** [comment]
Hey @aromalanil 
Let me know your feedback. We can do some changes too.

File: README.md [hunk 1]
```diff
@@ -19,27 +19,17 @@ npm install react-rhino
 yarn add react-rhino
 ```
 
-## Why?
```
Comment (aromalanil) [inline README.md:22]: I prefer to keep the Why section in the top itself

File: README.md [hunk 2]
```diff
@@ -19,27 +19,17 @@ npm install react-rhino
...
-Add a new global state with just a single line of code, it's that simple.
+### Getting started
+Set up state management for your application following these three easy steps shown in the example. Or see a  [demo project](https://codesandbox.io/s/react-rhino-example-svv5b) showing `react-rhino` in use.
 
-### 📐 Easy to Use
-Learn how to use Rhino in a short amount of time.
 
-## Online Playground
-[Here](https://codesandbox.io/s/react-rhino-example-svv5b) is a demo project for you to get familiar with React Rhino.
+1. ### Create global state
```
Comment (aromalanil) [inline README.md]: I see how a step by step instruction would do better, but here the hierarchy of the Heading tags are not correct .

eg: Both "Getting started" & "Create global state" are h3 tags

File: README.md [hunk 3]
```diff
@@ -19,27 +19,17 @@ npm install react-rhino
...
+To get started, create a file called `states.js`. This file holds a declaration of the global state for the entire app.
 
-## Example
-Here is an example of how to use React Rhino
 ### A file to create global states
```
Comment (aromalanil) [inline README.md]:
```suggestion
```

I think this line is no more needed

File: README.md [hunk 4]
```diff
@@ -130,11 +131,36 @@ const Toggle = () => {
...
+  const addressDetails = useRhinoValue("addressDetails")
+  return(<div>
+      <span>{userName}</span>
+  </div>
+  )
+
+}
+```
+
+## Online Playground
```
Comment (aromalanil) [inline README.md]: I think `Online Playground` deserves a better position at the top of the document

File: README.md [hunk 5]
```diff
@@ -150,7 +176,7 @@ const { RhinoProvider, useRhinoState } = createRhinoState({
 export { RhinoProvider, useRhinoState }
 ```
 
-> `createRhinoState` will output `RhinoProvider` and a bunch of hooks. Destructure the Provider and the hooks you need.
+> `createRhinoState` will output `RhinoProvider` and `useRhinoState`, `useRhinoValue` and `useSetRhinoState` hooks. Destructure the `createRhinoSate` Provider to get these  hooks as needed(like in the above snippet).
```
Comment (aromalanil) [inline README.md]:
```suggestion
> `createRhinoState` will output `RhinoProvider` , `useRhinoState`, `useRhinoValue` and `useSetRhinoState` hooks. 
```

File: README.md [hunk 6]
```diff
@@ -179,21 +205,38 @@ const [darkMode, setDarkMode] = useRhinoState("isDarkMode");
...
 /* Here "isDakMode" is the key representing the state */
 ```
 
 ### useSetRhinoState
 Takes key representing state object as input and returns the function to update the state.
-> You can use this if your component only wants the updater function and not the state itself.
-Having an updater function in the component will not trigger a rerender on the state change 
+> You can use this if your component only needs the updater function and not the state itself.
+Having an updater function in the component will not trigger a rerender on the state change.
+Updator functions perform updates on the state values.
```
Comment (aromalanil) [inline README.md]:
```suggestion
```

**Comment by aromalanil** [comment]
Hey @naftalimurgor 
Sorry for the delay. My exams are going on 😅

I have added some suggestions, go through each suggestion and change them if we share the same thoughts, if not comment your opinion.

**Comment by naftalimurgor** [comment]
The suggestions are perfectly great. Will work on them. Btw, I haven't used markdown that much. I wish you success in your exams ! 😃

**Comment by naftalimurgor** [comment]
Hey @aromalanil 
I made the changes per the suggestions we had. I also added License type at the bottom :smile: Let me know if there are other enhancements we can add.
Thanks!

**Comment by aromalanil** [comment]
@naftalimurgor Thanks for the changes. Just a few more small corrections and this will be merged.

1. Installation should come just before why section.
2. There should only be 3 steps in the getting started section. Others can be added as separate heading under usage section with same hierarchy as Getting started

**Comment by naftalimurgor** [comment]
@aromalanil 
Great! I'll make the changes to the mentioned sections.

**Comment by aromalanil** [comment]
LGTM 🌷

Congrats @naftalimurgor on your first merge to React Rhino
Celebrate

**Comment by aromalanil** [comment]
LGTM 🌷

Congrats @naftalimurgor on your first merge to React Rhino
Celebrate

**Pull request merged**